content/en/02-installing-pixie/01-requirements.md

```diff
@@ -69,23 +62,41 @@ Pixie runs on Linux nodes only.
 | Linux   | Supported       | v4.14+            |
 | Windows | Not Supported   | Not in roadmap    |
 
-### Linux Distributions
+### Linux Distribution
```

Comment:
same feedback here - let's try to keep unrelated changes in a separate diff, it's harder to review bigger diffs that have a lot going on